lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1417,6 +1451,13 @@ def test_basic(self, mock_emit):
             "group_name": None,
             "author": self.user.username,
             "author_label": None,
+            "author_image_url": {
+                "image_url_full": "http://testserver/static/default_500.png",
+                "image_url_large": "http://testserver/static/default_120.png",
+                "image_url_medium": "http://testserver/static/default_50.png",
+                "image_url_small": "http://testserver/static/default_30.png",
+                "has_image": False
```

Comment:
The default dictionary is being used in several places, it would be nice if we can define this once.